roughness-ref.csv

```diff
@@ -14,8 +14,8 @@ group,land cover,z0,zd,ra,reference
 7-1,,,,,
 7-2,,,,,
 8-1,,,,,
-8-2,,,,,
+8-2,Grasslands,2.6cm,10cm,0.4scm-1,https://www.jstor.org/stable/pdf/2402780.pdf?refreqid=excelsior%3Ac1e513dc450b5d460a8d2a27df982864
```

Comment:
Could you please modify the reference to an academic style? e.g., `Smith et al. (2019) <your link>`

Reply:
Okay on that now